index.bs

```diff
@@ -2825,23 +2798,6 @@ partial interface MLGraphBuilder {
     1. Return |output|.
 </details>
 
-#### {{MLGraphBuilder/gelu()}} #### {#api-mlgraphbuilder-gelu}
-<div>
-    **Arguments:**
-        - None.
-
-    **Returns:**
-        - an {{MLActivation}}. The activation function representing the gelu operation.
-</div>
-
-<details open algorithm>
-  <summary>
-    The <dfn method for=MLGraphBuilder id=gelu-noargs>gelu()</dfn> method steps are:
-  </summary>
-    1. Let |op| be the result of [=creating an MLActivation=] given [=this=] and "gelu".
-    1. Return |op|.
-</details>
-
```

Comment:
Appears ELU, CELU, RELU, *GELU* are supported with DML RNN's including RNN, LSTM, GRU. I don't really care about it (because I don't know of Gelu being used within that context), but know it's not "definitely not useable" 😉.

(the other two, clamp and softmax definitely are not though)

![image](https://github.com/webmachinelearning/webnn/assets/1809166/f1d3e593-da62-4627-86bb-df674348ddcc)

[dml_rnn_activation_test.json](https://github.com/user-attachments/files/15709938/dml_rnn_activation_test.json)